lib/ansible/executor/module_common.py

```diff
@@ -50,6 +55,69 @@
 
 # ******************************************************************************
 
+ZIPLOADER_DATA = u'''%(shebang)s
+# -*- coding: utf-8 -*-'
+import os
+import sys
+import base64
+import tempfile
+
+ZIPDATA = """%(zipdata)s"""
+
+def debug(command, zipped_mod):
+    # Okay to use __file__ here because we're running from a kept file
+    basedir = os.path.dirname(__file__)
+    if command == 'expand':
```

Comment:
It works fine to collapse the expand and execute into one shot- makes interactive debugging easier. We could still support them independently too, but the pycharm thing works a lot easier if it's all in one.